functionary/ui/templates/core/scheduledtask_detail.html

```diff
@@ -1,77 +1,63 @@
-{% extends "base_bulma.html" %}
+{% extends "base.html" %}
 {% load static %}
 {% block content %}
-    <nav class="breadcrumb has-arrow-separator mt-3" aria-label="breadcrumbs">
-        <ul>
-            <li>
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <li class="breadcrumb-item">
                 <a href="{% url 'ui:scheduledtask-list' %}">Schedules List</a>
             </li>
-            <li class="is-active">
-                <a href="#">{{ scheduledtask.name }}</a>
-            </li>
-        </ul>
+            <li class="breadcrumb-item active" aria-current="page">{{ scheduledtask.name }}</li>
+        </ol>
     </nav>
-    <div class="block mt-4">
-        <h1 class="title is-1">
-            <p>
-                <i class="fa fa-clock"></i>
-                <span>{{ scheduledtask.name }}</span>
-                <a href="{% url 'ui:scheduledtask-update' scheduledtask.id %}">
-                    <button class="button is-small has-text-link is-white singletonActive">
-                        <span class="fa fa-pencil-alt"></span>
-                    </button>
-                </a>
-            </p>
+    <div class="hstack gap-3 pt-2 pb-4">
+        <h1>
+            <i class="fa fa-clock"></i>
+            {{ scheduledtask.name }}
         </h1>
+        <i class="fa fa-pencil-alt fa-lg text-info singletonActive"
```

Comment:
What is `singletonActive`? Is that a holdover from bulma or is it doing something here?

Reply:
Okay, that probably makes sense then.  Carry on.